clash_lib/src/config/internal/convert/mod.rs

```diff
@@ -93,9 +90,7 @@ pub(super) fn convert(mut c: def::Config) -> Result<config::Config, crate::Error
                     OutboundProxy::ProxyServer(OutboundProxyProtocol::try_from(x)?);
                 let name = proxy.name();
                 if rv.contains_key(name.as_str()) {
-                    return Err(Error::InvalidConfig(format!(
-                        "duplicated proxy name: {name}"
-                    )));
+                    return Err(anyhow!("duplicated proxy name: {name}"));
```

Comment:
i think we should limit the usage of anyhow! it's quite flexible and hard to track error types

Reply:
Another PR, refacor the whole error type need too much efforts

Reply:
Then just leave the current untouched.